common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -22,6 +24,19 @@ object FaciaContentFrontendHelpers {
       imageOverride.orElse(defaultTrailPicture)
     }
 
+    def mainYouTubeMediaAtom: Option[MediaAtom] =
+      for {
+       main <- faciaContent.properties.maybeContent.map(_.fields.main)
+       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       document <- Some(Jsoup.parse(main))
```

Comment:
I am aware it's pretty ghastly to scrape HTML like this 👻  but as I couldn't manage to add the `blocks` to the content model without compilation errors to do with implicits, this is another way I can derive the information. At least it is encapsulated here so if we can get a more semantic way to work we can swap it out.  CC @TBonnin